improver_tests/nbhood/circular_kernel/test_CircularNeighbourhood.py

```diff
@@ -136,48 +117,47 @@ def test_multi_point_multitimes(self):
             expected[0][8 + index][8:13] = slice_
         for index, slice_ in enumerate(SINGLE_POINT_RANGE_3_CENTROID):
             expected[1][5 + index][5:10] = slice_
-        ranges = 3
-        result = CircularNeighbourhood(weighted_mode=True).apply_circular_kernel(
-            cube, ranges
-        )
+        radius = 6000
+        result = Neighbourhood("circular", weighted_mode=True).run(cube, radius)
         self.assertArrayAlmostEqual(result.data, expected)
 
     def test_single_point_masked_to_null(self):
-        """Test behaviour with a masked non-zero point. The behaviour here is
```

Comment:
Before the scipy code didn't handle masked data, so this test produced a result that was wrong. Now the masked data is handled by reusing the code from the square kernel, and this test returns a correct result. The same is true of the test below.